### ISSUE TITLE
Update 12 -> 13 ; Stuck in step 10 : Move new files in place : Parsing response failed

### ISSUE THREAD

**Issue by Feratti** [nextcloud/updater#156]
Hi guys!

After upgrading from v12 to v13 of nextcloud, i faced with problem in step 10 Move new files in place : Parsing response failed

When i make manually "php updater.phar" i got the error : 
<b>Parse error</b>:  syntax error, unexpected '<' in <b>../nextcloud/updater/updater.phar</b> on line <b>38</b><br />

Shared server provider is Godaddy

This is updater.log output:

```
2018-02-13T23:19:22-0700 TrIjMpo8kl [info] storage location: /home/content/03/11942903/html/nextcloud/data/updater-oca3vflrdijq/downloads/
2018-02-13T23:19:23-0700 TrIjMpo8kl [info] end of verifyIntegrity()
2018-02-13T23:19:23-0700 TrIjMpo8kl [info] endStep("6")
2018-02-13T23:19:23-0700 ZCvCajCuaE [info] request to updater
2018-02-13T23:19:23-0700 ZCvCajCuaE [info] currentStep()
2018-02-13T23:19:23-0700 ZCvCajCuaE [info] Step 6 is in state "end".
2018-02-13T23:19:23-0700 ZCvCajCuaE [info] POST request for step "7"
2018-02-13T23:19:24-0700 ZCvCajCuaE [info] startStep("7")
2018-02-13T23:19:24-0700 ZCvCajCuaE [info] extractDownload()
2018-02-13T23:19:24-0700 ZCvCajCuaE [info] storage location: /home/content/03/11942903/html/nextcloud/data/updater-oca3vflrdijq/downloads/
2018-02-13T23:20:10-0700 ZCvCajCuaE [info] end of extractDownload()
2018-02-13T23:20:10-0700 ZCvCajCuaE [info] endStep("7")
2018-02-13T23:20:10-0700 6k9okdrkil [info] request to updater
2018-02-13T23:20:10-0700 6k9okdrkil [info] currentStep()
2018-02-13T23:20:10-0700 6k9okdrkil [info] Step 7 is in state "end".
2018-02-13T23:20:10-0700 6k9okdrkil [info] POST request for step "8"
2018-02-13T23:20:11-0700 6k9okdrkil [info] startStep("8")
2018-02-13T23:20:11-0700 6k9okdrkil [info] replaceEntryPoints()
2018-02-13T23:20:11-0700 6k9okdrkil [info] replace index.php
2018-02-13T23:20:11-0700 6k9okdrkil [info] replace status.php
2018-02-13T23:20:11-0700 6k9okdrkil [info] replace remote.php
2018-02-13T23:20:11-0700 6k9okdrkil [info] replace public.php
2018-02-13T23:20:11-0700 6k9okdrkil [info] replace ocs/v1.php
2018-02-13T23:20:11-0700 6k9okdrkil [info] replace ocs/v2.php
2018-02-13T23:20:11-0700 6k9okdrkil [info] end of replaceEntryPoints()
2018-02-13T23:20:11-0700 6k9okdrkil [info] endStep("8")
2018-02-13T23:20:11-0700 eJ3xvtWENh [info] request to updater
2018-02-13T23:20:11-0700 eJ3xvtWENh [info] currentStep()
2018-02-13T23:20:11-0700 eJ3xvtWENh [info] Step 8 is in state "end".
2018-02-13T23:20:11-0700 eJ3xvtWENh [info] POST request for step "9"
2018-02-13T23:20:11-0700 eJ3xvtWENh [info] startStep("9")
2018-02-13T23:20:11-0700 eJ3xvtWENh [info] deleteOldFiles()
2018-02-13T23:20:18-0700 eJ3xvtWENh [info] config sample exists
2018-02-13T23:20:18-0700 eJ3xvtWENh [info] thmes README exists
2018-02-13T23:20:39-0700 eJ3xvtWENh [info] end of deleteOldFiles()
2018-02-13T23:20:39-0700 eJ3xvtWENh [info] endStep("9")
```

**Comment by Feratti** [comment]
![error](https://user-images.githubusercontent.com/36470026/36210044-deb98dc8-11b6-11e8-9e68-8f7b5eb92505.PNG)

**Comment by Feratti** [comment]
Problem was solved with command rsync -Aax nextcloud_newversion/ nextcloud/

**Comment by MorrisJobke** [comment]
> Parse error: syntax error, unexpected '<' in ..../nextcloud/updater/updater.phar on line 38

:/ Not nice ... but I can't reproduce this,

**Comment by prschopf** [comment]
same issue here, 13.0.2 > 13.0.3, shared hosting

What I did to solve it:

mv nextcloud nextcloud.old
wget https://download.nextcloud.com/server/releases/nextcloud-13.0.3.zip
unzip nextcloud-13.0.3.zip
cp nextcloud.old/config/config.php nextcloud/config
changed maintenance mode to false
Refreshed update page so, I went directly to database update procedure
changed maintenance mode to false

Now, its updated. After this procecure, you may clean some files: rm -fr nextcloud.old nextcloud-13.0.3.zip

**Comment by phirschbeck** [comment]
same issue here, 13.0.2 > 13.0.6

Solved it with a simply click on "retry update"!

**Comment by nichierichetti** [comment]
Hello all,

I just had this problem in updating to 21.0.7
`Move new files in place. Parsing response failed.`

the detailed response is simply the same content of the update page but in un-parsed html

**Comment by nichierichetti** [comment]
> Hello all,
> 
> I just had this problem in updating to 21.0.7 `Move new files in place. Parsing response failed.`
> 
> the detailed response is simply the same content of the update page but in un-parsed html

UP

**Comment by joshtrichards** [comment]
The *Parsing Response Failed* was fixed in #90.

@Feratti Not sure why things originally failed, but may be related to #519 or other oddities with some hosting environments. The inability to run the phar is weird, but maybe it was corrupted somehow. Glad the manual update approach worked!

@phirschbeck The need to do a *Retry Update* (once or several times) _may_ be addressed with pending work on the Iterator in (#519). 

Since this is such a old report and many things have changed, I'm going to close this out. But any of you feel free to create new dedicated issues for any new bugs you encounter. Thanks for your reports!
